install_config/aggregate_logging.adoc

```diff
@@ -70,6 +70,15 @@ region or other selector you would like to use to restrict where the
 logging infrastructure deploys.
 ====
 +
+[NOTE]
+====
+Restricting the project will also restrict where Fluentd can be deployed.
+It is instead recommended to specify node selectors on your Kibana and
+Elasticsearch Deployment Configs once created.
+
+See xref:node_selector[Node Selector] for examples on how to do this.
```

Comment:
Use `link:` for links. See [here](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references).